From 1c59ae0b5ca747be8f5253edc26c5582fec53b44 Mon Sep 17 00:00:00 2001
From: Minecrell <minecrell@minecrell.net>
Date: Wed, 6 Jan 2021 12:51:48 +0100
Subject: [PATCH 36/78] drm/panel: samsung-s6e88a0-ams427ap24: Add
 mipi_dsi_dcs_set_display_on()

The display does not show up otherwise.
---
 .../msm8916-generated/panel-samsung-s6e88a0-ams427ap24.c    | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drivers/gpu/drm/panel/msm8916-generated/panel-samsung-s6e88a0-ams427ap24.c b/drivers/gpu/drm/panel/msm8916-generated/panel-samsung-s6e88a0-ams427ap24.c
index c14a8ecb3..a489a0901 100644
--- a/drivers/gpu/drm/panel/msm8916-generated/panel-samsung-s6e88a0-ams427ap24.c
+++ b/drivers/gpu/drm/panel/msm8916-generated/panel-samsung-s6e88a0-ams427ap24.c
@@ -87,6 +87,12 @@ static int s6e88a0_ams427ap24_on(struct s6e88a0_ams427ap24 *ctx)
 	dsi_dcs_write_seq(dsi, 0xf0, 0xa5, 0xa5);
 	dsi_dcs_write_seq(dsi, 0xfc, 0xa5, 0xa5);
 
+	ret = mipi_dsi_dcs_set_display_on(dsi);
+	if (ret < 0) {
+		dev_err(dev, "Failed to set display on: %d\n", ret);
+		return ret;
+	}
+
 	return 0;
 }
 
-- 
2.31.1

